pages/linux/dbus-daemon.md

```diff
@@ -0,0 +1,31 @@
+# dbus-daemon
+
+> The D-Bus message daemon, allowing multiple programs to exchange messages.
+
+- Run the daemon with a configuration file:
+
+`dbus-daemon --config-file {{path/to/file}}`
+
+- Run the daemon with the standard per-login-session message bus configuration:
+
+`dbus-daemon --session`
+
+- Run the daemon with the standard systemwide message bus configuration:
+
+`dbus-daemon --system`
+
+- Set the address to listen on and ignore the configuration for this:
+
+`dbus-daemon --address {{address}}`
+
+- Output the process id to stdout:
+
+`dbus-daemon --print-pid`
+
+- Force the message bus to write to the system log for messages:
+
+`dbus-daemon --syslog`
+
+- Disable the creation of a pid file and ignore the configuration for this:
```

Comment:
Is this a common enough use-case ? I believe usually you would want to use a pid file ? (Also, same question as above)

Reply:
I guess probably not really. 🤔

Reply:
Sure, I'll remove it when I get to work. :+1: